calabash-cucumber/calabash-cucumber.gemspec

```diff
@@ -59,7 +59,7 @@ Gem::Specification.new do |s|
   # Match the xamarin-test-cloud dependency.
   s.add_dependency('bundler', '~> 1.3')
   s.add_dependency("clipboard", "~> 1.0")
-  s.add_dependency("run_loop", ">= 3.0", "< 4.0")
+  s.add_dependency("run_loop", ">= 3.0", "< 5.0")
```

Comment:
Let's update this to ">= 4.0".

Reply:
I wasn't sure if we want to restrict users to really use only 4+